src/SimpleTable.types.ts

```diff
@@ -47,6 +47,10 @@ export type Props<K extends IndexType> = {
   headerRenderer?(column: Column): string | Renderable
   bodyRenderer?(row: Row<K>, columnID: K): string | Renderable
 
+  // dynamic CSS classes
+  headerCellClass?(column: Column): string
+  bodyCellClass?(row: Row<K>, columnID: K): string
```

Comment:
Could we have the docs you wrote in the pull request description here as documentation?